package.json

```diff
@@ -74,19 +74,21 @@
     "lib/",
     "plugins/",
     "index.d.ts",
-    "react-markdown.min.js",
-    "with-html.d.ts",
-    "with-html.js"
+    "react-markdown.min.js"
   ],
   "dependencies": {
-    "@types/mdast": "^3.0.3",
+    "@types/hast": "^2.3.0",
     "@types/unist": "^2.0.3",
```

Comment:
the types here have more specific versions than `^2.0.0`.
Is this intended? IIRC you prefer wider version ranges.

Reply:
I think I opted for more stricter versions because I don’t know about the changelog of `@types/hast`, and also that some other `@types/` packages here are stricter.
I think it can be made wider